packages/fluentui/react-northstar/src/themes/teams/components/TextArea/textAreaStyles.ts

```diff
@@ -34,6 +35,13 @@ export const textAreaStyles: ComponentSlotStylesPrepared<TextAreaStylesProps, Te
       color: v.disabledColor,
     }),
 
+    ...(p.autoAdjustHeight && {
+      minHeight: v.minHeightAutoAdjust,
+      height: pxToRem(p.autoHeight),
```

Comment:
Please could you move this `height` to inline style? If I understand correctly - each change in `autoHeight` here would trigger styles change in the document head, which would be more costly performance-wise, than change of the inline style

Reply:
done